_tutorials/page3.md

```diff
@@ -1,12 +1,101 @@
 ---
 layout: tutorial
-title: Tutorial
-name: Chaining requests
-previous_page_title: Creating custom templates
+title: Run
+name: Run
+previous_page_title: API Sign Up
+next_page_title: Tests
 ---
 
-## Part 3
+# Run ScanAPI
 
-Duis congue porttitor blandit. Fusce euismod orci in enim rhoncus, vitae porta sapien sagittis. Integer vitae est sit amet sapien vulputate accumsan ac eu dui. Suspendisse quis elit vitae quam tincidunt tincidunt vel nec tellus. Integer ornare erat eget libero mattis porta. Donec mollis tortor ante, sit amet pretium risus pharetra faucibus. Maecenas tellus enim, commodo at ligula nec, sollicitudin auctor eros. Donec consectetur leo urna, vitae sollicitudin dolor ullamcorper sodales. Aliquam id nisi lorem. Fusce tincidunt congue libero, in cursus erat auctor sed. Suspendisse potenti. Curabitur efficitur, est et sollicitudin pellentesque, leo dolor feugiat turpis, et faucibus nunc quam vitae justo. Etiam id nibh mauris. Donec molestie auctor dapibus. Quisque vehicula laoreet justo et tristique.
+It’s time to make ScanAPI work! Create a new directory for ScanAPI files and name it whatever you
+want. Through the rest of this tutorial we’ll refer to this directory as **root**.
 
-Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Donec vel nisi quis orci mollis molestie ac porta tellus. Vivamus arcu quam, sollicitudin vitae lorem sed, malesuada venenatis lorem. Proin nec tristique lorem. Pellentesque pulvinar pellentesque lacus eget molestie. In porttitor egestas lorem vitae tempus. Suspendisse ultrices justo vestibulum ex tempus, non interdum lectus dapibus. Ut eu pulvinar nunc. Morbi aliquam nisi nec vestibulum venenatis.
+For example:
+
+```shell
+$ mkdir scanapi
```

Comment:
Is this a reference to the person's project? If so, it would be cool to make it clear what is the expected structure in their project.

```
- root (your project root)
  - my_folder_a
  - my_folder_b
  - scanapi.yml
```

Reply:
@george-silva at this point of the tutorial, it is just testing an external API, not related to an own project. Later on in the tutorial, I will create a page called "Add ScanAPI to your project". In this page, I will add instructions on how to add ScanAPI using circleci/github actions.

But either way, I can still add the expected folder structure, even thought it will be quite simple for now.

```
- root
  - scanapi.yaml 
``` 

Thanks for the idea!